app/component/Itinerary.js

```diff
@@ -237,7 +237,7 @@ const hasOneTransitLeg = data => {
   return data.legs.filter(leg => leg.transitLeg).length === 1;
 };
 
-const SummaryRow = (
+const Itinerary = (
```

Comment:
Maybe call this ItinerarySummary to differentiate that this isn't the details page? Again, there are some classes etc that use the old naming.

Reply:
There is already another ItinerarySummary component, where name makes sense. It is shown on top of itinerary details.

Reply:
And the actual legs are rendered by a component called ItineraryLegs which is pretty OK.